src/views/private/admin-leaderboard/addTeam.js

```diff
@@ -98,6 +102,7 @@ function AddTeam(props) {
 
 
     const modalStyle = {
```

Comment:
Can we make all the modal boxes not rounded so it matches everything else since nothing else is rounded?

Reply:
Deleted all of the default border radius!

Reply:
Awesome, thanks! Looks great